### ISSUE TITLE
Gears schematics improvement

### ISSUE THREAD

**Issue by benallard** [jessecrossen/ttsim#11]
Hi, I'd like to suggest a better schematics for the gears, instead of relying on the color of the arrow to distinguish the "bit" and the "geared bit".

You could simply represent the gear as a circle, so the the red gear would simply be a circle, and the geared bit, an arrow with a circle on top of it.

What do you think ?

I could probably drop you a merge-request for it if you agree.

**Comment by jessecrossen** [comment]
Yes, I had wanted to do something like that, and tried several variations without finding one I liked. The hard part is to indicate the circular geometry without obscuring the outline so much that the arrow direction isn't clear, and this becomes especially important when zoomed out, which is where schematic mode is most useful. I think the physical device has a similar problem with distinguishing the direction of gearbits, but generally one stands close enough to touch it so the problem isn't too bad. Of course with a schematic representation, we don't have the same constraints. I'd love to see some sketches of what you're thinking!
